src/app/components/ItemFilters/ItemFilters.jsx

```diff
@@ -17,95 +14,66 @@ const ItemFilters = (
     displayDateFilter,
     numOfFilteredItems,
     itemsAggregations = [],
-    dispatch,
     numItemsTotal,
     numItemsCurrent,
-    mappedItemsLabelToIds = {}
+    fieldToOptionsMap = {}
   },
   { router },
 ) => {
   const mediaType = React.useContext(MediaContext);
   const { createHref, location } = router;
   const query = location.query || {};
+  // there are multiple location codes for the single label offsite, 
+  //   to keep count and maintain state for the Offsite location filter,
+  //   we need to combine those filters back into one string
+  const initialLocations = (locations) => {
+    const concatenatedRecapLocations = locations.filter((loc) => loc.startsWith('loc:rc')).join(',')
```

Comment:
It'd be good to point out what the `'loc:rc'` prefix means.

Reply:
👍